demo/api/src/products/product-category.resolver.ts

```diff
@@ -0,0 +1,58 @@
+import { AffectedEntity, CometValidationException, RequiredPermission } from "@comet/cms-api";
+import { InjectRepository } from "@mikro-orm/nestjs";
+import { EntityManager, EntityRepository } from "@mikro-orm/postgresql";
+import { Args, ID, Int, Mutation, Resolver } from "@nestjs/graphql";
+
+import { ProductCategory } from "./entities/product-category.entity";
+import { ProductCategoriesService } from "./generated/product-categories.service";
+
+@Resolver(() => ProductCategory)
+@RequiredPermission(["products"], { skipScopeCheck: true })
+export class ProductCategoryResolver {
+    constructor(
+        private readonly entityManager: EntityManager,
+        private readonly productCategoriesService: ProductCategoriesService,
+        @InjectRepository(ProductCategory) private readonly repository: EntityRepository<ProductCategory>,
+    ) {}
+
+    @Mutation(() => Boolean)
+    @AffectedEntity(ProductCategory)
+    async updateProductCategoryPosition(
```

Comment:
Did you think about returning the entity instead of boolean? This would make the API more similar to create/update

Reply:
I thought about this option but kept boolean because my draft provided by @johnnyomair also returned true. But maybe this makes sense because it's possible to query `productCategory { products { id, position } }` so it should probably be possible to work somehow with apollo cache?

Reply:
ok, no I mixed something. I can remember I didn't switch to ProductCategory because this mutation does alter other product-category entities as well. so probably returning all changed entities would be appropriate but would then make this operation use much more memory (if working with entities instead of nativeUpdate) or more complicated (if optionally fetching changed rows, but that would probably be ok)